src/index.jsx

```diff
@@ -130,6 +131,7 @@ function render(Component) {
     loadUserInfo: isGoogleAuthority,
     userStore: new WebStorageStateStore({ store: window.localStorage }),
     extraQueryParams: { access_type: 'offline' },
+    revokeTokensOnSignout: true,
```

Comment:
Cool - doing this in Terra UI as well :)

Reply:
Nice :)